Gemfile.lock

```diff
@@ -170,6 +170,7 @@ GEM
     websocket-driver (0.7.1)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.4)
+    yard (0.9.19)
```

Comment:
Name: yard
Version: 0.9.19
Advisory: 
Criticality: Unknown
URL: https://github.com/lsegal/yard/security/advisories/GHSA-xfhh-rx56-rxcr
Title: Possible arbitrary path traversal and file access via `yard server`
Solution: Upgrade to >= 0.9.20.